android/src/main/java/com/robinpowered/Frequency/FrequencyTrackFactory.java

```diff
@@ -56,4 +50,24 @@ public static AudioTrack create(double frequency, double duration) {
 
         return track;
     }
+
+    static byte[] createSoundData(double frequency, int numberOfSamples) {
+        double sample[] = new double[numberOfSamples];
+        byte soundData[]= new byte[2 * numberOfSamples];
+
+        for (int i = 0; i < numberOfSamples; ++i) {
```

Comment:
Is there a particular reason for `++i` here? If not, let's do `i++`. We aim to put it before the number only when intentional